data/en/imagecaptcha.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "imageCaptcha",
+	"type": "function",
+	"syntax": "imageCaptcha()",
+	"returns": "any",
+	"related": ["imagewritetobrowser"],
+	"description": "Provides information about the time when the current application scope was created.",
+	"params": [
+            {"name":"text","description":"Text in the image captcha.","values":[],"default":"","type":"string","required":true},
+            {"name":"height","description":"Height of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"width","description":"Width of the image captcha.","values":[],"default":"","type":"numeric","required":true},
+            {"name":"difficulty","description":"Width of the image captcha.","values":[],"default":"medium","type":"string","required":false},
+            {"name":"fonts","description":"One or more fonts for the CAPTCHA text, as an array or as string list (separated by commas). Lucee supports the system fonts recognized by JDK only.","values":[],"default":"arial","type":"any","required":false},
+            {"name":"fontsize","description":"Font size for text of the image captcha.","values":[],"default":"24","type":"any","required":false},
+            {"name":"fontcolor","description":"Font color for text of the image captcha.","values":[],"default":"black","type":"string","required":false}
+  ],
+	"engines": {
+		"lucee": {"minimum_version": "5", "notes": "", "docs": "https://docs.lucee.org/reference/functions/imagecaptcha.html"}
```

Comment:
It looks like the minimum version might be 5.3 based on some searching. Can anyone verify?